<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QMainWindow" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="resultTextBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>470</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="authorization" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>271</width>
      <height>81</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="login">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>0</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="password">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="SIgnIn">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Sign In</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="mainForm" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>371</width>
      <height>181</height>
     </rect>
    </property>
    <widget class="QPushButton" name="LoadVideo">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>load video</string>
     </property>
    </widget>
    <widget class="QLabel" name="fileName">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>fileName</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SendVideo">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>send video</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="userHistory">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="seeHistory">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>History</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="videoResult">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="adminFrame">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>371</width>
      <height>161</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <widget class="QLineEdit" name="newLogin">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="newPassword">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addUser">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add User</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="selectRigths">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>121</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>user rigths</string>
     </property>
     <widget class="QRadioButton" name="simpleUser">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>simple user</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="advancedUser">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>advanced user</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="admin">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>administrator</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="advancUserFrame">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>361</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QPushButton" name="loadPictures">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Picturse</string>
     </property>
    </widget>
    <widget class="QLabel" name="loadedPicture1">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>0</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addPictures">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Pictures</string>
     </property>
    </widget>
    <widget class="QLabel" name="loadedPicture2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="loadedPicture3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>485</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
